src/ood-gen/bin/watch_scrape.ml

```diff
@@ -1,76 +1,124 @@
 open Piaf
+open Lwt_result.Syntax
 
-type t = {
+type watch = {
   name : string;
   embedPath : string;
   thumbnailPath : string;
-  description : string;
-  year : string;
+  description : string option;
+  date : string;
   language : string;
   category : string;
 }
 
-(* Handling Ezjsonm.get_string Parse error *)
-let get_string_from_value json =
-  try Ezjsonm.get_string json
-  with Ezjsonm.Parse_error (`Null, "Ezjsonm.get_string") -> ""
+type t = { watch : watch list }
 
 (* extract year from originally published date string *)
 let get_year json =
-  let s = Ezjsonm.get_string json in
-  String.sub s 0 4
+  match Ezjsonm.find json [ "originallyPublishedAt" ] with
```

Comment:
Unrelated to this PR, but is there a reason for using `Ezjsonm` instead of `Yojson`? (I'm under the impression that the later is more mainstream and stable, but maybe that's just me)

Reply:
Realised I should also change the name of this function 🤦 ... Two reasons sorted by importance for `Ezjsonm`:

1. Ezjsonm's `value` is identical to Yaml's value (by design). This (at least for me) makes it easier to reason about the two given they have the same underlying structure. I don't mind switching to `Yojson` though.  
2. Given the people currently working on ood, we have a little more control over Ezjsonm (given it's a mirage project) :))